test/performance/specs/front-end-block-theme.spec.js

```diff
@@ -19,19 +14,27 @@ test.describe( 'Front End Performance', () => {
 		await requestUtils.activateTheme( 'twentytwentythree' );
 	} );
 
-	test.afterAll( async ( { requestUtils } ) => {
-		saveResultsFile( __filename, results );
+	test.afterAll( async ( { requestUtils }, testInfo ) => {
+		await testInfo.attach( 'results', {
+			body: JSON.stringify( results, null, 2 ),
+			contentType: 'application/json',
+		} );
 		await requestUtils.activateTheme( 'twentytwentyone' );
 	} );
 
-	test( 'Report TTFB, LCP, and LCP-TTFB', async ( { page } ) => {
-		let i = 16;
-		while ( i-- ) {
+	test( 'Report TTFB, LCP, and LCP-TTFB', async ( { browser } ) => {
+		const samples = 16;
+		const throwaway = 0;
+		const rounds = samples + throwaway;
+
+		for ( let i = 0; i < rounds; i++ ) {
+			const testPage = await browser.newPage( { storageState: {} } );
+
 			// Go to the base URL.
-			await page.goto( '/', { waitUntil: 'networkidle' } );
+			await testPage.goto( '/', { waitUntil: 'networkidle' } );
```

Comment:
> DISCOURAGED consider operation to be finished when there are no network connections for at least 500 ms. Don't use this method for testing, rely on web assertions to assess readiness instead.

I guess we still want this despite the docs discouraging it?

Reply:
Yeah, I'm aware but unsure what to replace it with. Now that I'm looking at it, maybe we can just remove it as in [this example](https://www.checklyhq.com/learn/headless/basics-performance/#:~:text=Largest%20Contentful%20Paint%20API%20(largest%2Dcontentful%2Dpaint))?